pkg/operator/controller/status/controller.go

```diff
@@ -411,7 +442,7 @@ func computeOperatorDegradedCondition(ingresses []operatorv1.IngressController)
 			case operatorv1.ConditionTrue:
 				degradedCondition.Status = configv1.ConditionTrue
 				degradedCondition.Reason = "IngressDegraded"
-				degradedCondition.Message = fmt.Sprintf("The %q ingress controller reports Degraded=True: %s: %s", ic.Name, cond.Reason, cond.Message)
+				degradedCondition.Message = fmt.Sprintf("The %q ingress controller reports Degraded=True: %s: %s.", ic.Name, cond.Reason, cond.Message)
```

Comment:
We'll need to make sure we never change `computeIngressDegradedCondition` to add a period at the end.  * sigh *, * grin *.